backend/src/db/migrations/20230820083309_add_user_details_table.ts

```diff
@@ -0,0 +1,43 @@
+import { type Knex } from 'knex';
+
+import { DatabaseTableName } from '#libs/packages/database/database.js';
```

Comment:
Reduce the imports to migrations

Reply:
Sorry, what does reduce the imports mean?  Am I supposed do telete one? I just use DatabaseTableName in this file

Reply:
Migrations should be independent of any deps like enums, types etc. Bcos if we decide to change for example DatabaseTableName migration will fail and we can't change it

Reply:
Ok, got it